lib/tasks/deploy.rake

```diff
@@ -2,6 +2,7 @@ GITHUB_ORG = 'ad2games'
 BASE_IMAGE = 'docker-rails:latest'
 DEPLOY_USER = 'ad2gamesdeploy'
 DEPLOY_EMAIL = 'developers@ad2games.com'
+DEPLOY_ENVS = { 'staging' => %w(staging) } # N.b., master is not deployed by default
```

Comment:
In some cases we still don't have a preview branch.  I guess we could still set the default to both and override for the exceptions, though.